### PR TITLE
chore(bench): run groups of benchmarks using env variable

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: tfhe/benches/integer/bench.rs [hunk 1]
```diff
@@ -773,48 +773,62 @@ criterion_group!(
 );
 
 criterion_group!(
-    arithmetic_parallelized_operation,
+    default_ops,
```
Comment (IceTDrinker) [inline tfhe/benches/integer/bench.rs:776]: default_parallelized_ops maybe ?

File: tfhe/benches/integer/bench.rs [hunk 2]
```diff
@@ -773,48 +773,62 @@ criterion_group!(
...
-    smart_scalar_arithmetic_parallel_operation,
+    smart_scalar_parallelized_ops,
     smart_scalar_add_parallelized,
     smart_scalar_sub_parallelized,
     smart_scalar_mul_parallelized,
 );
 
 criterion_group!(
-    scalar_arithmetic_parallel_operation,
+    default_scalar_ops,
```
Comment (IceTDrinker) [inline tfhe/benches/integer/bench.rs:814]: missing parallelized

File: tfhe/benches/shortint/bench.rs [hunk 3]
```diff
@@ -638,33 +640,33 @@ define_server_key_scalar_div_bench_fn!(
...
-    unchecked_mul_msb,
-    unchecked_div,
+    smart_ops,
     smart_bitand,
     smart_bitor,
     smart_bitxor,
     smart_add,
     smart_sub,
     smart_mul_lsb,
+    programmable_bootstrapping
```
Comment (IceTDrinker) [inline tfhe/benches/shortint/bench.rs:650]: no need to add it here it's already in the unchecked ops and it's a kind of unchecked op so fine like this !

File: tfhe/benches/integer/bench.rs [hunk 4]
```diff
@@ -849,52 +863,27 @@ criterion_group!(
...
-    scalar_left_shift_parallelized,
-    scalar_right_shift_parallelized,
-    scalar_eq_parallelized,
-    scalar_ne_parallelized,
-    scalar_lt_parallelized,
-    scalar_le_parallelized,
-    scalar_gt_parallelized,
-    scalar_ge_parallelized,
-    scalar_min_parallelized,
-    scalar_max_parallelized,
```
Comment (IceTDrinker) [inline tfhe/benches/integer/bench.rs:888]: did you check we cover the same functions as before ?

Comment (soonum) [reply tfhe/benches/integer/bench.rs:888]: Yes, I've just rearranged the current coverage into separate groups.

Comment (IceTDrinker) [reply tfhe/benches/integer/bench.rs:888]: ok thanks !

**Review by IceTDrinker — APPROVED** [review]
Looks good thanks !

**Pull request merged**